### PR TITLE
Contributing Guide: "How You Can Help"

### PR DESCRIPTION
**Pull request by BD103**
This adds "How You Can Help" from [HackMD](https://hackmd.io/blGjQ_xUSIGRv-gsePsDsg), as well as some minor tweaks to style. Check out the commits for a full list of changes (though there aren't many!).

### CODE REVIEW

**Review by NthTensor — APPROVED** [review]
I drafted/edited portions of this, so interpret this approval as you will.

We kind of went crazy with the callouts in this section, and could walk them back if people feel strongly about it. But I like how they separate direct instructions from the more informative sections.

File: content/contributing/helping-out/how-you-can-help.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+[Reddit]: https://www.reddit.com/r/bevy
+[itch.io]: https://itch.io/games/tag-bevy
+
+## Reviewing Others' Work
+
+**One of the most valuable things you can do is review other people's work.** Bevy relies on community code review to keep the code quality high and the maintenance burden low. Absolutely anyone is welcome to leave reviews, and all reviews are treated with equal weight.
+
+You don't need to be an Elder Rustacean to be useful here: anyone can catch missing tests, unclear docs, logic errors, and so on. If you're new to Rust or Bevy, submitting reviews is great way to learn things and acquaint yourself with the internals of the engine.
+
+If you do have specific skills (e.g. advanced familiarity with `unsafe` code, rendering knowledge, or web development experience) or personal experience with a problem, that's fantastic! We suggest trying to prioritize those areas, to ensure we can get appropriate expertise where we need it.
```
Comment (mweatherley) [inline content/contributing/helping-out/how-you-can-help.md:34]:
```suggestion
If you do have specific skills (e.g. advanced familiarity with `unsafe` code, rendering knowledge, or web development experience) or experience with a specific problem, that's fantastic! We suggest trying to prioritize those areas to ensure we get appropriate expertise where we need it.
```

File: content/contributing/helping-out/how-you-can-help.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+Not even our Project Leads and Maintainers are exempt from reviews! By giving feedback on this work (and related supporting work), you can help us make sure our releases are both high-quality and timely. 
+
+{% callout() %}
+Interested in reviewing but don't know where to start? Check out [Reviewing Pull Requests](TODO)!
+{% end %}
+
+## Joining a Working Group
+
+Bevy's active initiatives are organized into *temporary working groups*: public, open-membership teams where people work together to tackle a broad-but-scoped goal. Each working group coordinates through a dedicated forum-channel on [Discord], although they also create issues and may use project boards for tangible work that needs to be done.
```
Comment (mweatherley) [inline content/contributing/helping-out/how-you-can-help.md:44]:
```suggestion
Bevy's active initiatives are organized into *temporary working groups*: public, open-membership teams where people work together to tackle a sizeable, well-scoped goal. Each working group coordinates through a dedicated forum-channel on [Discord], but they may also create issues or use project boards to organize and track their progress.
```
I don't know what "broad-but-scoped" is supposed to convey; maybe that the projects are still somewhat substantial? I tried to convey that.

File: content/contributing/helping-out/how-you-can-help.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+You should consider joining a working group if you're interested in contributing code but don't know where to start or what to work on. Choosing one and asking how to help can be a fantastic way to get up to speed and be immediately useful.
```
Comment (mweatherley) [inline content/contributing/helping-out/how-you-can-help.md:46]:
```suggestion
You should consider joining a working group if you're interested in contributing but don't know where to start or what to work on. Choosing one and asking how to help can be a fantastic way to get up to speed and be immediately useful.
```
I don't think it has to be contributing code specifically, right? Of course, that's obviously welcome :P

File: content/contributing/helping-out/how-you-can-help.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+There are no special requirements to participate in a working group, as well as no application, formal membership list, or leadership. Anyone can help, and you should expect to compromise and work together with others to bring a shared vision to life. Working groups are _spaces_, not clubs.
```
Comment (mweatherley) [inline content/contributing/helping-out/how-you-can-help.md:48]:
```suggestion
There are no special requirements to participate in a working group — no applications, formal membership list, or even leadership. Anyone can help, and you should expect to compromise and work together with others to bring your shared vision to life. Working groups are _spaces_, not clubs.
```

File: content/contributing/helping-out/how-you-can-help.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+{% callout() %}
+Anyone can start working group! Check out [Working Groups](TODO) for more information about the process and requirements.
```
Comment (mweatherley) [inline content/contributing/helping-out/how-you-can-help.md:51]:
```suggestion
Anyone can start a working group! Check out [Working Groups](TODO) for more information about the process and requirements.
```

File: content/contributing/helping-out/how-you-can-help.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+{% end %}
+
+## Contributing Code
+
+We love getting code contributions from the community, and there's plenty of work to go around! If you want to code but don't know what to work on, you should take a look at the open issues. The page on [Triage](@/contributing/reference/triage.md) details several tags that you can use to filter issues by difficulty and area.
+
+If you already know what you want to work on, make your changes and submit a pull request! The page on [Your First Pull Request](TODO) is there to help if you get stuck.
```
Comment (mweatherley) [inline content/contributing/helping-out/how-you-can-help.md:58]:
```suggestion
If you already know what you want to work on, all you have to do is make your changes and submit a pull request! The page on [Your First Pull Request](TODO) is there to help if you get stuck.
```

File: content/contributing/helping-out/how-you-can-help.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+Check out our [plugin guidelines](@/learn/quick-start/plugin-development.md) for helpful tips and patterns!
+{% end %}
+
+## Organizing Issues and Pull Requests
+
+If nothing brings you more satisfaction than seeing every last issue labeled and all resolved issues closed, feel free to hop on [Discord] and ask `@Maintainer` to be added to the Github organization. Anyone interested in helping us keep things neat and tidy is welcome to join. As will be discussed later, this role only requires good faith, a basic understanding of our development process, and a few merged pull requests.
+
+## Teaching Others
+
+Bevy is still very young, and light on documentation, tutorials, and accumulated expertise. By helping others with their issues and teaching them about Bevy, you will naturally learn the engine and codebase in greater depth (while also making our community better)!
```
Comment (mweatherley) [inline content/contributing/helping-out/how-you-can-help.md:86]:
```suggestion
Bevy is still very young, and it's light on documentation, tutorials, and accumulated expertise. By helping others with their issues and teaching them about Bevy, you will naturally learn the engine and codebase in greater depth (while also making our community better)!
```

**Review by mweatherley — COMMENTED** [review]
Just some random suggestions for improving wording in some places

File: content/contributing/helping-out/how-you-can-help.md [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+[GitHub Discussions]: https://github.com/bevyengine/bevy/discussions
+[Discord]: https://discord.gg/bevy
+[Reddit]: https://www.reddit.com/r/bevy
+[itch.io]: https://itch.io/games/tag-bevy
+
+## Reviewing Others' Work
+
+**One of the most valuable things you can do is review other people's work.** Bevy relies on community code review to keep the code quality high and the maintenance burden low. Absolutely anyone is welcome to leave reviews, and all reviews are treated with equal weight.
+
+You don't need to be an Elder Rustacean to be useful here: anyone can catch missing tests, unclear docs, logic errors, and so on. If you're new to Rust or Bevy, submitting reviews is great way to learn things and acquaint yourself with the internals of the engine.
```
[the remark this answers is not in the document]
Comment (BD103) [reply content/contributing/helping-out/how-you-can-help.md:32]: I'm not sure I agree with this suggestion. It turns the entire paragraph into one run-on sentence, when in reality it is two distinct statements.

File: content/contributing/helping-out/how-you-can-help.md [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+title = "How You Can Help"
+insert_anchor_links = "right"
+[extra]
+weight = 1
++++
+
+If you've made it to this page, you're probably already convinced that Bevy is a project you'd like to see thrive.
+But how can *you* help?
+
+No matter how experienced you are with Bevy and Rust or what your level of commitment is, there is a way for you to contribute! Take a look at the sections below to find a route that appeals to you (or many!).
```
Comment (BD103) [inline content/contributing/helping-out/how-you-can-help.md:11]:
```suggestion
No matter how experienced you are with Bevy and Rust, or what your level of commitment is, there is a way for you to contribute! Take a look at the sections below to find a route that appeals to you (or many!).
```

**Review by alice-i-cecile — APPROVED** [review]

**Pull request merged**